packages/cloud-cognitive/src/components/Datagrid/Datagrid/addons/CustomizeColumns/Columns.js

```diff
@@ -54,6 +57,73 @@ const Columns = ({
     },
     [columns, setColumnsObject]
   );
+
+  const setColumnStatus = (id, isVisible) => {
+    let list = document.querySelectorAll(
+      `li.${blockClass}__draggable-handleHolder`
+    );
+    for (let i = 0; i < list.length; i++) {
+      if (columns[i].id.includes(id)) {
+        if (isVisible) {
+          list[i].setAttribute('selected', 'true');
+        } else {
+          list[i].setAttribute('selected', 'false');
+        }
+      }
```

Comment:
Perhaps try using array functions. This could end up looking something like

```
list.forEach((li, i) => {
       if(!columns[i].id.includes(id)) continue
       li[i].setAttribute('selected', isVisible)
})
```